### DISCUSSION TITLE
How to filter Relationship field data in a form?

### DISCUSSION THREAD

**Discussion by arladmin** [Budibase/budibase#6669] [Help]
I would like to be able to filter the Relational data options visible in a Relationship Picker component, in a form?

I tried incorporating a Data Provider and adding an Options Picker component in it. That didn't really work out though.

**Comment by magickaito** [comment]
You can look at workaround posted at https://github.com/Budibase/budibase/discussions/5661.

**Reply by arladmin** [reply]
Thanks for your message.
Although, i'm already aware of this, and my question is specifically with regards to the Relationship Picker in a Form, and not a table.

**Comment by melohagan** [comment]
Hi @arladmin 

The options for the Relationship Picker cannot be filtered. They are automatically set to be *all* of the possible related rows.
